test/WebJobs.Script.Tests/Environment/EnvironmentTests.cs

```diff
@@ -149,6 +149,30 @@ public void IsPersistentStorageAvailable_IsLinuxWithStorage_ReturnsTrue()
             Assert.True(environment.IsPersistentFileSystemAvailable());
         }
 
+        [Fact]
+        public void IsKubernetesAppService_NoKubernetesServiceHost_ReturnsFalse()
```

Comment:
recommend moving these tests to EnvironmentExtensionsTests class, and using a single [Theory] test as in other examples to test all the permutations.